L123-core-add-alts-google-call-credentials.md

```diff
@@ -0,0 +1,83 @@
+# Support ALTS Credentials in Google Default Credentials
```

Comment:
Let's replace ALTS credentials with ALTS Hard Bound Call Credentials. ALTS Credentials would be confusing since there is already the ALTS channel credentials.